README.md

```diff
@@ -92,17 +92,100 @@ Run `npm run e2e:build:android && npm run e2e:test:android` to build the APK for
 
 ## Translations
 
-### Adding new text
+### Adding and changing new source strings
+
+Source strings are in `src/i18n/strings.ftl` and should be in US English. Don't edit the files in `src/i18n/l10n/` because they will get overwritten when we pull in translations. All translation is done by volunteers on [Crowdin](https://crowdin.com/editor/inaturalistios/724), so please follow these guidelines to make things easier for those generous people.
+
+1. **Labels should match content as closesly as possible** (without exceeding 100 characters)
+    * Bad
+        ```fluent
+        collection-project-screen-title = ABOUT COLLECTION PROJECTS
+        ```
+
+    * Good
+        ```fluent
+        ABOUT-COLLECTION-PROJECTS = ABOUT COLLECTION PROJECTS
+        ```
+
+1. **Labels should change when the content changes**
+    * Bad
+        * Old
+            ```fluent
+            ABOUT-COLLECTION-PROJECTS = ABOUT COLLECTION PROJECTS
+            ```
+
+        * New
+            ```fluent
+            ABOUT-COLLECTION-PROJECTS = ABOUT COLLECTION AND UMBRELLA PROJECTS
+            ```
+
+    * Good
+        * Old
+            ```fluent
+            ABOUT-COLLECTION-PROJECTS = ABOUT COLLECTION PROJECTS
+            ```
+
+        * New
+            ```fluent
+            ABOUT-COLLECTION-AND-UMBRELLA-PROJECTS = ABOUT COLLECTION AND UMBRELLA PROJECTS
+            ```
+
+1. **Annotate strings with comments** unless the string is very self-descriptive
+    * Bad
+        ```fluent
+        Change-date = Change date
+        ```
+
+        Is this a verb phrase or a noun phrase? Are we talking about spare change in your pocket?
+
+    * Good
+        ```fluent
+        # Label for a button that changes a selected date
+        Change-date = Change date
+        ```
+
+1. **Use double-dashes to append extra context to keys and to keep them unique and descriptive.** For example, translators might need to translate the word "Unknown" differently if it refers to a place or a taxon, so you might include both `Unknown--place = Unknown` and `Unknown--taxon = Unknown`
+1. **Accessibility hints** are used by screen readers to describe what happens
+when the user interacts with an element. The [iOS Guidelines](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619585-accessibilityhint) define it as "A string that briefly describes the result of performing an action on the accessibility element." We write them in third person singular ending with a period
+1. **Pluralize text with a count** using [selectors](https://projectfluent.org/fluent/guide/selectors.html)
+    * Bad
+        ```fluent
+        x-observations = { $count } observations
+        ```
+
+    * Good
+        ```fluent
+        x-observations = { $count } { $count ->
+          [one] observation
```

Comment:
are there any guidelines on when we need [some] and [few]? we seem to not be using those right now

Reply:
I need some input from translators on this. In `strings.ftl`, we only *need* the plural forms that American English requires, so `[one]` and `*[other]`, and translators can choose to support whatever plural forms they want in their languages. However, this code is fairly complex and I'm not sure we can assume translators will just know what all the available options are in Fluent, so I'm considering advising (or maybe even requiring) us to use all the forms in `strings.ftl` to provide a template for translators, even though most of the plural forms will be unnecessary in most languages.